test/cpp/end2end/xds_end2end_test.cc

```diff
@@ -2167,25 +2182,45 @@ TEST_P(LdsTest, WrongRouteSpecifier) {
             AdsServiceImpl::NACKED);
 }
 
+using LdsRdsTest = BasicTest;
+
+// Tests that LDS client should send an ACK upon correct LDS response (with
+// inlined RDS result).
+TEST_P(LdsRdsTest, Vanilla) {
+  if (GetParam().enable_rds_testing()) {
```

Comment:
Instead of putting these three lines in each individual test, why not do this in `XdsEnd2endTest::SetUp()` right where we create each balancer?